bin/publish-docs.sh

```diff
@@ -5,12 +5,12 @@
 
 PACKAGE_VERSION=$(node -e 'console.log(require("./package").version)')
 
+git fetch origin gh-pages
 git checkout gh-pages
-git pull origin gh-pages
 git merge -
 npm run public-docs
 git rm -r docs
 mv out docs
 git add docs
 git commit -m "docs: update for ${PACKAGE_VERSION}"
-git push origin gh-pages && git push public gh-pages:gh-pages
\ No newline at end of file
+git push origin gh-pages
```

Comment:
👍 

The original script assumed you were running locally on a repo who's origin pointed to GitHub Enterprise and had a remote of 'public' pointing to the public repo mirror. Now we are just working off of public GitHub and are assuming running on a clean pull of the repo.